lib/utils/atRuleRegex.mjs

```diff
@@ -0,0 +1,9 @@
+import { nestingSupportedAtKeywords } from '../reference/atKeywords.mjs';
+
+export const MEDIA_REGEX = /^media$/i;
+export const KEYFRAMES_REGEX = /^(-(o|moz|ms|webkit)-)?keyframes$/i;
```

Comment:
off-topic

@jeddy3 @ybiquitous 
Should `keyframes-name-pattern` use this regex?

https://github.com/stylelint/stylelint/blob/8ef1792d08755282b74216ead1508af94556538d/lib/rules/keyframes-name-pattern/index.mjs#L32

ref #1437

Reply:
Great catch. I think we should use this regex for the rule, too. :+1:

Reply:
Since it's part of the test it can be considered a refactor.
i.e. could be done in this PR

Reply:
Good suggestion! I've fixed in the following commit.
[Refactor to use regexes](https://github.com/stylelint/stylelint/pull/8272/commits/25c8276b8dbaf6eea9aba96d7d77d40bf6e85e7d)